image_to_table/extractor/image_with_word_boxes.py

```diff
@@ -37,23 +35,23 @@ def merge_sorted_text_boxes(boxes: List[TextBox]) -> TextBox:
     return TextBox(description, bounding_rect)
 
 
-def extract_table_from_image(filename: str, placement: List[int]) -> List[List[TextBox]]:
-    image = cv2.imread(filename)
+def extract_table_from_image(content: bytes) -> List[List[str]]:
+    image = cv2.imdecode(np.frombuffer(content, np.uint8), 1)
```

Comment:
Pretty sure reading from bytes is very quick, but you're absolutely right, can definitely rather read once in the parent function and pass the image!

Reply:
ah btw, need to do this here, because we call this method from the endpoint. And we send the bytes to Google anyway, so we can't send them an actual image

Reply:
but if you mean in the two separate functions, good point, I'll change that

Reply:
Well, meant maybe both. But I'll settle with one of the two 😁